doc/missions.md

```diff
@@ -0,0 +1,133 @@
+# Missions
+
+## MissionSimplePayment
+
+Basic test to complete a payment on a small network of nodes.
+
+## MissionComplexTopology
+
+Complete basic payment load generation on an asymmetric network topology.
+
+## MissionInMemoryMode
+
+Complete basic payment and Soroban load generation on a network with in-memory watcher nodes.
+
+## MissionLoadGeneration
+
+Complete extended payment load generation (sustained tx rate) on a small network of nodes.
+
+## MissionLoadGenerationWithSpikes
+
+Complete extended payment load generation (sustained tx rate with periodic spikes) on a small network of nodes.
+
+## MissionLoadGenerationWithTxSetLimit
+
+Complete extended payment load generation while triggering surge pricing on a small network of nodes.
+
+## MissionMixedImageLoadGeneration
+
+Complete extended payment load generation while triggering surge pricing with mixed version images on a small network of nodes. Tests both old image majority and new image majority.
+
+## MissionHistoryGenerateAndCatchup
+
+Test catchup to generated ledger state for each protocol version.
+
+## MissionHistoryPubnetMinimumCatchup
+
+Perform minimum catchup to Public Network. Runs core `catchup current/0` command  under the hood.
```

Comment:
Tiny nitpick: There are two spaces between "command" and "under" here and in a few other places in this file. Seems to just be a copy/paste error.

Reply:
good catch, fixed.